### ISSUE TITLE
Read the entire consumerconfig from the appsettings?

### ISSUE THREAD

**Issue by oTcom** [SorenZ/Alamut.Kafka#1]
[https://github.com/SorenZ/Alamut.Kafka/blob/master/src/Alamut.Kafka/KafkaSubscriber.cs](url)

```
var config = new ConsumerConfig
            {
                ClientId =_kafkaConfig.ClientId,
                EnableAutoCommit = _kafkaConfig.EnableAutoCommit,
                SaslUsername = _kafkaConfig.SaslUsername ,
                EnablePartitionEof = _kafkaConfig.EnablePartitionEof,
                StatisticsIntervalMs = _kafkaConfig.StatisticsIntervalMs,
                GroupId = _kafkaConfig.GroupId,
                SaslMechanism = _kafkaConfig.SaslMechanism,
                BootstrapServers = _kafkaConfig.BootstrapServers,
                AutoOffsetReset = _kafkaConfig.AutoOffsetReset,
                SaslPassword = _kafkaConfig.SaslPassword,
                SecurityProtocol = _kafkaConfig.SecurityProtocol
            };
```

**Comment by SorenZ** [comment]
good idea.
consider that it's in a very early version (even not beta) and under heavy changes.
anyway, I welcome any PR related to the issue or I will fix it in the near future.
Many thanks.

**Comment by SorenZ** [comment]
@oTcom 
Problem solved in [v0.7.0](https://github.com/SorenZ/Alamut.Kafka/tree/v0.7.0).
Now you can put everything in the `KafkaConfig`  of `appsettings.json` and it will bind it to the `ConsumerConfig` or `ProducerConfig`.
